ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -104,6 +146,7 @@ void RSkComponentTextInput::onHandleKey(rnsKey eventKeyType, bool *stopPropagati
   *stopPropagation = false;
   cursor_.end = textString.length();
   auto component = getComponentData();
+  auto const &textInputProps = *std::static_pointer_cast<TextInputProps const>(component.props);
```

Comment:
instead of reading the props here , please create a class member as editable and update this value in updateProps function.